tendrl/common/atoms/base_atom.py

```diff
@@ -0,0 +1,11 @@
+import abc
+
+
+class BaseAtom(object):
+    __metaclass__ = abc.ABCMeta
+
+    @abc.abstractmethod
```

Comment:
BaseAtom class should contain below attributes as per Tendrl schema 0.3 (https://raw.githubusercontent.com/Tendrl/node_agent/master/tendrl/node_agent/manager/tendrl_definitions_node_agent.py)
- name (str)
- enabled (bool)
- inputs (set)
- help (str) (requires change to current definitions, change current "name" to "help")
- outputs (set)
- uuid (str)